internal/deviations/deviations.go

```diff
@@ -99,4 +107,21 @@ var (
 	StaticProtocolName = flag.String("deviation_static_protocol_name", "DEFAULT", "The name used for the static routing protocol.  The default name in OpenConfig is \"DEFAULT\" but some devices use other names.")
 
 	DeprecatedVlanID = flag.Bool("deviation_deprecated_vlan_id", false, "Device requires using the deprecated openconfig-vlan:vlan/config/vlan-id or openconfig-vlan:vlan/state/vlan-id leaves.")
+
+	ExplicitInterfaceInVRF = flag.Bool("deviation_explicit_interface_in_vrf", true,
```

Comment:
That's correct, I somehow missed that one. Thanks for pointing that out!

```suggestion
	ExplicitInterfaceInVRF = flag.Bool("deviation_explicit_interface_in_vrf", false,
```